notebooks/stable-diffusion-v2/stable-diffusion-v2-torch-compile.ipynb

```diff
@@ -0,0 +1,217 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stable Diffusion v2 for Text-to-Image Generation\n",
+    "\n",
+    "To start, let's look on Text-to-Image process for Stable Diffusion v2. We will use [Stable Diffusion v2-1](https://huggingface.co/stabilityai/stable-diffusion-2-1) model for these purposes. The main difference from Stable Diffusion v2 and Stable Diffusion v2.1 is usage of more data, more training, and less restrictive filtering of the dataset, that gives promising results for selecting wide range of input text prompts. More details about model can be found in [Stability AI blog post](https://stability.ai/blog/stablediffusion2-1-release7-dec-2022) and original model [repository](https://github.com/Stability-AI/stablediffusion).\n",
+    "\n",
+    "### Stable Diffusion in Diffusers library\n",
+    "To work with Stable Diffusion v2, we will use Hugging Face [Diffusers](https://github.com/huggingface/diffusers) library. To experiment with Stable Diffusion models, Diffusers exposes the [`StableDiffusionPipeline`](https://huggingface.co/docs/diffusers/using-diffusers/conditional_image_generation) similar to the [other Diffusers pipelines](https://huggingface.co/docs/diffusers/api/pipelines/overview).  The code below demonstrates how to create `StableDiffusionPipeline` using `stable-diffusion-2-1`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "05bdfee9ce504db69b64d4ba5dd4451e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Loading pipeline components...:   0%|          | 0/6 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/user/Downloads/openvino_notebooks/.venv/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "from diffusers import StableDiffusionPipeline\n",
+    "import torch \n",
+    "import numpy as np\n",
+    "from transformers import CLIPTokenizer\n",
+    "from diffusers.schedulers import DDIMScheduler\n",
+    "\n",
+    "seed = 42\n",
+    "\n",
+    "np.random.seed(seed)\n",
+    "torch.manual_seed(seed)\n",
+    "\n",
+    "pipe = StableDiffusionPipeline.from_pretrained(\"stabilityai/stable-diffusion-2-1-base\").to(\"cpu\")\n",
+    "\n",
+    "pipe.text_encoder = torch.compile(pipe.text_encoder.eval(), backend='openvino')\n",
+    "pipe.unet = pipe.unet.eval()\n",
```

Comment:
why excluding unet ？

Reply:
That was for a different experiment, I did not push the updated notebook. I have done it now.